.github/workflows/publish-image.yml

```diff
@@ -36,6 +36,7 @@ jobs:
       - name: Build and publish a Docker image for ${{ github.repository }}
         uses: macbre/push-to-ghcr@master
         with:
+          #dockerfile: Docker/Dockerfile
```

Comment:
this part may actually have to be uncommented, otherwise the action may look for `Dockerfile` in the repository root location and not the `Docker/` folder.

you can check these changes work or result in error by manually triggering the action.

you can do that by visiting the [actions](https://github.com/abheektripathy/urNode/actions/workflows/publish-image.yml) tab in your fork and you will see a button "Run workflow" from which you can manually trigger the docker build action

![image](https://user-images.githubusercontent.com/39442192/199228014-d3b27137-7510-46c2-bfb3-6f3d1182a94c.png)